ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-hosted-pipeline.yml

```diff
@@ -41,6 +41,9 @@ spec:
       default: quay-oauth-token
     - name: quay_oauth_secret_key
       default: token
+    - name: developer
```

Comment:
This flag is not too self- explanatory- what do you think about naming it something like `skip_hydra_check`?

Reply:
I'd make it even more descriptive. Hydra is quite a large service with many components. How about `ignore_publishing_checklist` instead? We also don't want to skip the query but rather ignore the results.